src/test/java/com/znsio/sample/e2e/screen/web/amazonsearch/AmazonHomeScreenWeb.java

```diff
@@ -0,0 +1,48 @@
+package com.znsio.sample.e2e.screen.web.amazonsearch;
+
+import com.context.TestExecutionContext;
+import com.znsio.e2e.runner.Runner;
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.screen.amazonsearch.AmazonHomeScreen;
+import com.znsio.sample.e2e.screen.amazonsearch.AmazonSearchResultsScreen;
+import org.apache.commons.lang.NotImplementedException;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class AmazonHomeScreenWeb extends AmazonHomeScreen{
+    private final Driver driver;
+    private final Visual visually;
+    private static final String SCREEN_NAME = AmazonHomeScreenWeb.class.getSimpleName();
+    private static final Logger LOGGER = Logger.getLogger(SCREEN_NAME);
+
+    public AmazonHomeScreenWeb(Driver driver, Visual visually) {
+        super();
+        this.driver = driver;
+        this.visually = visually;
+        visually.checkWindow(SCREEN_NAME, "Home Page");
+    }
+
+    public static final By bySearchBoxXpath = By.xpath("//input[@id='twotabsearchtextbox']");
```

Comment:
if id is present then why to go with xpath

Reply:
refactored